pandas/tests/reshape/test_reshape.py

```diff
@@ -11,8 +11,8 @@
 
 from pandas.util.testing import assert_frame_equal
 
-from pandas.core.reshape.reshape import (
-    melt, lreshape, get_dummies, wide_to_long)
+from pandas.core.reshape.reshape import get_dummies
```

Comment:
use ``from pandas import get_dummies, melt, lreshape, wide_to_long``. 

in tests we always want to use the user call (rather than a specific import)